pkg/expr/nodes.go

```diff
@@ -418,16 +417,23 @@ func getResponseFrame(resp *backend.QueryDataResponse, refID string) (data.Frame
 	return response.Frames, nil
 }
 
-func convertDataFramesToResults(ctx context.Context, frames data.Frames, datasourceType string, s *Service, logger log.Logger) (string, mathexp.Results, error) {
+func ConvertDataFramesToResults(ctx context.Context,
+	frames data.Frames,
+	datasourceType string,
+	features featuremgmt.FeatureToggles,
+	allowLongFrames bool,
+	tracer tracing.Tracer,
+	logger log.Logger,
```

Comment:
This is a pretty long arg list for exporting. Maybe something like `c := NewFrameToResultCoverter(...)` and then `c.Convert(..` ?